pallets/difficulty/src/tests.rs

```diff
@@ -159,8 +166,14 @@ fn exercise_on_timestamp_set_when_previous_is_configured() {
 			})
 			.unwrap();
 		assert_eq!(previous.len(), 2);
+		PreviousDifficultiesAndTimestamps::<Test>::put(previous);
 
+		TargetBlockTime::<Test>::put(60000); // ms
+		DifficultyAdjustmentPeriod::<Test>::put(100i64);
 		<timestamp::Pallet<Test>>::set(Origin::none(), 98765).unwrap();
+
+		// just exercise the getter function
+		let _ = crate::Pallet::<Test>::difficulty();
```

Comment:
This will always be exercisable, did you mean to unwrap the output?

Reply:
Nope. The return value is `sp_core::U256`. I just meant to execute the getter function without making a useless assert like the some of the above examples.